ubuntucleaner/janitor/firefox_history_plugin.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+import glob
+
+from ubuntucleaner.settings.debug import get_traceback, run_traceback
+from ubuntucleaner.janitor import CacheObject, JanitorPlugin
+
+log = logging.getLogger('Janitor')
+
+class FirefoxDataPlugin(JanitorPlugin):
+    __title__ = _('Firefox Browsing Data')
+    __category__ = 'application'
+
+    def __init__(self):
+        self.profile_location = os.path.expanduser("~/.mozilla/firefox")
+        JanitorPlugin.__init__(self)
+
+    def get_cruft(self):
```

Comment:
Can't you achieve the same but extending from JanitorCachePlugin and define targets? So inside the root_path it will look for defined targets.